test/fixtures/application.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const jstp = require('../../');
+
+const name = 'testApp';
+const login = 'login';
+const password = 'password';
+const sessionId = '12892e85-5bd7-4c77-a0c5-a0aecfcbc93a';
+
+const interfaces = {};
+
+const authCallback = (
+  connection, application, strategy, credentials, callback
+) => {
+  if (application.name !== name) {
+    return callback(new jstp.RemoteError(jstp.ERR_APP_NOT_FOUND));
+  }
+
+  let username = null;
+  let success = false;
+
+  if (strategy === 'anonymous') {
+    success = true;
+  }
+
+  if (strategy === 'login' &&
+    credentials[0] === login &&
+    credentials[1] === password) {
+    success = true;
+    username = login;
+  }
+
+  if (success) {
```

Comment:
In my opinion this part would look better written like this:
```
if (!success) {
  return callback(new jstp.RemoteError(jstp.ERR_AUTH_FAILED));
}
callback(null, username, sessionId);
```
Just to remove extra indentation and to be more consistent with the style usually used in Node.js.